$NetBSD: patch-aa,v 1.4 2012/01/09 17:57:34 wiz Exp $

Separate build of xmms plugin from the rest, so it can be split in two packages.

--- configure.orig	2004-08-26 22:16:19.000000000 +0000
+++ configure
@@ -20632,6 +20632,7 @@ CFLAGS="$CFLAGS $X_CFLAGS -Wall"
 M4DATADIR="$datadir/aclocal"
 
 
+if [ "$XOSD_XMMS_PLUGIN" = "YES" ]; then
 
 # Check whether --with-gtk-prefix or --without-gtk-prefix was given.
 if test "${with_gtk_prefix+set}" = set; then
@@ -20947,6 +20948,7 @@ echo "$as_me: WARNING: *** GTK+ >= 1.2.2
 
   rm -f conf.gtktest
 
+fi
 
 echo "$as_me:$LINENO: checking for XInitThreads in -lX11" >&5
 echo $ECHO_N "checking for XInitThreads in -lX11... $ECHO_C" >&6
